composables/__tests__/useUserAddresses.test.ts

```diff
@@ -0,0 +1,91 @@
+import Vue from "vue"
+import VueCompositionApi from "@vue/composition-api"
+
+import { getUserAddressesQuery } from "@/lib/gql/queries"
+import { useUserAddresses } from "@/composables/useUserAddresses"
```

Comment:
export from index.ts. import should be from only @/composables

Reply:
Resolved.